lang/cs/Org.Apache.REEF.Client/Common/DriverFolderPreparationHelper.cs

```diff
@@ -52,7 +52,7 @@ internal sealed class DriverFolderPreparationHelper
         // We embed certain binaries in client dll.
         // Following items in tuples refer to resource names in Org.Apache.REEF.Client.dll
         // The first resource item contains the name of the file 
-        // such as "reef-bridge-java-0.13.0-incubating-SNAPSHOT-shaded.jar". The second resource
+        // such as "reef-bridge-java-0.14.0-incubating-SNAPSHOT-shaded.jar". The second resource
```

Comment:
Nit: Make this number <#N> or some placeholder. This will keep getting out of sync all the time.